schemas/telemetry/pioneer-study/pioneer-study.4.schema.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "properties": {
+    "creationDate": {
+      "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}\\.[0-9]{3}Z$",
+      "type": "string"
+    },
+    "id": {
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+      "type": "string"
+    },
+    "payload": {
+      "description": "",
+      "properties": {
+        "encryptedData": {
+          "description": "JOSE/JWE encrypted payload.",
+          "type": "string"
+        },
+        "encryptionKeyId": {
+          "description": "JOSE/JWK key id, e.g. pioneer-20170520.",
+          "type": "string"
```

Comment:
do you want a pattern on this?

Reply:
Not right now. I'll leave it up to ops to decide the naming scheme and don't want to presume anything.